configure.ac

```diff
@@ -52,31 +52,6 @@ dnl Allow 64-bit file API on 32-bit systems. Without the change even small
 dnl files will fail to stat any files on filesystems with 64-bit inodes.
 AC_SYS_LARGEFILE
 
-DEBUG=""
-AC_MSG_CHECKING([for debug-enabled build])
-AC_ARG_ENABLE(debug, AS_HELP_STRING([--enable-debug],[create a debug build]),
```

Comment:
This is the last patch that I did not apply so far.

I agree that we can remove this part here.